TypedSQL generates invalid JavaScript when used with driverAdapters

### Bug description

When `typedSQL` and `driverAdapters` are specified in previewFeatures, `index.edge.mjs` becomes `index.wasm.mjs` and the file contents ~and package.json~ are also wrong.

```js
// Correct Content
export * from "./getUsers.edge.mjs"

// Wrong Content
export * from "./getUsers.wasm.mjs"
```

### How to reproduce

<!--
1. Go to '...'
2. Change '....'
3. Run '....'
4. See error
-->

1. Pull https://github.com/miyaji255/prisma-typed-sql-file-name
2. `docker compose up -d`
3. `pnpm prisma generate --sql`
4. See prisma-client/sql/index.wasm.mjs


### Expected behavior

Output correct path.

### Prisma information


```prisma
generator client {
  provider        = "prisma-client-js"
  output          = "../prisma-client"
  previewFeatures = ["typedSql", "driverAdapters"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
}

```



### Environment & setup

- OS: Ubuntu 22
- Database: PostgreSQL
- Node.js version: v22.11.0

### Prisma Version

```
prisma                  : 6.4.1
@prisma/client          : 6.4.1
Computed binaryTarget   : debian-openssl-3.0.x
Operating System        : linux
Architecture            : x64
Node.js                 : v22.11.0
TypeScript              : unknown
Query Engine (Node-API) : libquery-engine a9055b89e58b4b5bfb59600785423b1db3d0e75d (at node_modules/.pnpm/@prisma+engines@6.4.1/node_modules/@prisma/engines/libquery_engine-debian-openssl-3.0.x.so.node)
Schema Engine           : schema-engine-cli a9055b89e58b4b5bfb59600785423b1db3d0e75d (at node_modules/.pnpm/@prisma+engines@6.4.1/node_modules/@prisma/engines/schema-engine-debian-openssl-3.0.x)
Schema Wasm             : @prisma/prisma-schema-wasm 6.4.0-29.a9055b89e58b4b5bfb59600785423b1db3d0e75d
Default Engines Hash    : a9055b89e58b4b5bfb59600785423b1db3d0e75d
Studio                  : 0.509.0
Preview Features        : driverAdapters, typedSql
```

This problem prevents `wrangler deploy`. Currently I simply replace `.wasm` with `.edge` after `prisma generate`, but I hope this will be resolved.

```
$ pnpm build:wrangler

> prisma-typed-sql-file-name@ build:wrangler /home/miyaji255/workspace/prisma-typed-sql-file-name
> wrangler deploy --outdir dist/workers --dry-run


 ⛅️ wrangler 3.114.0
--------------------


✘ [ERROR] Build failed with 1 error:

  ✘ [ERROR] Could not resolve "./getUsers.wasm.mjs"
  
      prisma-client-with-driverAdapters/sql/index.wasm.mjs:1:14:
        1 │ export * from "./getUsers.wasm.mjs"
          ╵               ~~~~~~~~~~~~~~~~~~~~~
  
  


🪵  Logs were written to "/home/miyaji255/.config/.wrangler/logs/wrangler-2025-03-07_12-56-07_855.log"
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/packages/client-generator-js/src/typedSql/typedSql.ts
+++ b/packages/client-generator-js/src/typedSql/typedSql.ts
@@ -34,7 +34,7 @@ export function buildTypedSql({
     fileMap[`${query.name}.js`] = buildTypedQueryCjs(options)
     fileMap[`${query.name}.${edgeRuntimeName}.js`] = buildTypedQueryCjs(edgeOptions)
     fileMap[`${query.name}.mjs`] = buildTypedQueryEsm(options)
-    fileMap[`${query.name}.edge.mjs`] = buildTypedQueryEsm(edgeOptions)
+    fileMap[`${query.name}.${edgeRuntimeName}.mjs`] = buildTypedQueryEsm(edgeOptions)
   }
   fileMap['index.d.ts'] = buildIndexTs(queries, enums)
   fileMap['index.js'] = buildIndexCjs(queries)

